client/src/components/MainContainer/DateSelector.tsx

```diff
@@ -0,0 +1,45 @@
+import useCurrentDate from '../../hooks/useCurrentDate';
+import styled from 'styled-components';
+import * as S from './Log.style';
+
+interface DateSelectArrowProps {
+  direction: string;
```

Comment:
유니온 타입을 적용하면 좋을 것 같습니다.
```suggestion
  direction: 'prev' | 'next';
```

Reply:
저도 이생각 했었는데 그냥 이렇게 바로 적용하면 되는군요?